src/traversal/diffRelayQuery.js

```diff
@@ -619,10 +619,12 @@ class RelayDiffQueryBuilder {
     // The returned `trackedNode` is never tracked directly: instead it serves
     // as an indicator to `diffConnection` that the entire connection field must
     // be tracked.
+    /* eslint-disable consistent-return */
```

Comment:
Let's just `return undefined` to make it explicit at the other site instead of disabling the rule